tests/functional/tests/basic/test_pyocf.py

```diff
@@ -32,7 +32,7 @@ def test_simple_wt_write(pyocf_ctx):
     core_device.reset_stats()
 
     write_data = Data.from_string("This is test data")
-    io = core.new_io(cache.get_default_queue(), 20, write_data.size,
+    io = core.new_io(cache.get_default_queue(), 512, write_data.size,
```

Comment:
You can use Size.from_sector(1).bytes (not sure about the exact syntax). Would be easier to support configurable sector size in the future.

Reply:
Great suggestion, thanks!